Mlem/App/Views/Root/Tabs/Settings/PostReadIndicatorSettingsView.swift

```diff
@@ -25,14 +25,10 @@ struct PostReadIndicatorSettingsView: View {
             SettingsHeaderView(
                 title: "Read Indicator",
                 // swiftlint:disable:next line_length
-                description: "When you've read a post already, its title will appear dimmed. If you like, you can choose an additional way of indicating read status."
-            ) {
-                Image(systemName: Icons.read)
-                    .resizable()
-                    .aspectRatio(contentMode: .fit)
-                    .frame(height: 60)
-                    .foregroundStyle(.tertiary)
-            }
+                description: "When you've read a post already, its title will appear dimmed. If you like, you can choose an additional way of indicating read status.",
```

Comment:
*nit* slightly more concise
```suggestion
                description: "Read posts are indicated with dimmed title text. If you like, you can choose an additional way of indicating read status.",
```